_data/ja/proposals.yml

```diff
@@ -0,0 +1,23 @@
+title: 現在の仕様候補一覧
+description: このセクションでは、<a href="https://tc39.es/process-document/">我々のプロセス</a>においてステージ3に到達している、つまり完了が近いプロポーザルを紹介しています。
+how-to-read:
+  title: プロポーザル一覧の読み方
+  description: 各プロポーザルは他の資料へのリンクが貼ってあります。表示されているタグは以下の属性を示しています。各タグが何を意味しているかは以下の例を参照ください。全てのタグはマウスカーソルをホバーさせたら詳細が表示されます。
+  tags:
+    title: タグの種類とそれぞれの意味
+    last-presented: 最終発表日
+    last-presented-fake-date: December 2018
```

Comment:
Is this forgotten, or was it intentional?

Reply:
I completely missed it, so I just pushed commit to fix it. You just beat me!